experimental/AlgebraicStatistics/src/CI.jl

```diff
@@ -0,0 +1,229 @@
+# -*- Conditional independence statements -*-
+
+export CIStmt, ci_stmt, @CI_str, ci_statements, make_elementary
+
+struct CIStmt
+  I::Vector{<:VarName}
+  J::Vector{<:VarName}
+  K::Vector{<:VarName}
+end
+
+@doc raw"""
+  ci_stmt(I::Vector{<:VarName}, J::Vector{<:VarName}, K::Vector{<:VarName}; symmetric=true, semigraphoid=true)
```

Comment:
```suggestion
@doc raw"""
    ci_stmt(I::Vector{<:VarName}, J::Vector{<:VarName}, K::Vector{<:VarName}; symmetric=true, semigraphoid=true)
```
The signature needs to be indented by exactly four spaces, otherwise it does not show correctly in the documentation. Try
```
build_doc(doctest=false)
```
to see the docs locally.

Reply:
This is the only occurrence of wrongly indented signature I found btw when building the docs locally, so unless you want to have a look at other parts of the documentation there is probably no need for you to build it locally.